sled-agent/src/bootstrap/params.rs

```diff
@@ -205,6 +217,8 @@ mod tests {
                         address: None,
                         mac: MacAddr6::nil().into(),
                     },
+                    ntp_servers: vec!["test.pool.example.com".parse().unwrap()],
+                    dns_servers: vec!["1.1.1.1".parse().unwrap()],
```

Comment:
These are just `&'static str` types, so to convert to a `String` type, you can call either:

`String::from("test.pool.example.com")` - https://doc.rust-lang.org/std/string/struct.String.html#impl-From%3C%26str%3E-for-String
or
`"test.pool.example.com".to_string()` - https://doc.rust-lang.org/std/string/trait.ToString.html#tymethod.to_string


`parse` makes sense if the conversion can fail, but since this conversion is infallible, we can use one of these alternatives to avoid the `unwrap`.